### ISSUE TITLE
need for colony distance parameters

### ISSUE THREAD

**Issue by VirginiaMorera** [BirdLifeInternational/track2kba#32]
Hello, 

I've been trying to follow the workflow in the vignette except that my trips are already split so I'm skipping that step. I find that `tripSummary` depends on the parameters calculated in the `tripSplit` funtion, and then `findScale` needs the summary to calculate some of the parameters, and then `estSpaceUse` needs those to specify a smoothing value. 

Not a huge issue, but it might be a good idea to specify which parameters are needed for the tripSummary function, and in what format, so people can jump in at any point in the workflow without having to go back to the start of the sequence!

**Comment by MartinBeal** [comment]
True. I'm looking into it. Let me know if you have specific suggestions.

**Comment by VirginiaMorera** [comment]
Ok, so I'm running through the vignette to see the output format at each step. `trips` is a spatial points dataframe, but the function `tripSummary` seems to use only the dataframe, ignoring the coordinates (since X and Y are columns in the dataframe itself). 

So, what if `tripSummary` has an option to receive a spdf (if it comes from the tripSplit) or a simple df (if the user starts with an already splitted dataframe) and then the help file specifies the columns that it needs? 

Will keep digging and try to come up with something more specific!

**Comment by MartinBeal** [comment]
I think the main barrier here is the Returns column. This is a nice feature of our trip-splitting method, but not one most people will have in their data frames. 

Perhaps we could change this to TRUE/FALSE (instead of strings) and users could just fill their own data sets with TRUE if they don't know/care. @steffenoppel any thoughts on this, considering you wrote these fxns?

**Comment by VirginiaMorera** [comment]
Otherwise, add a step to calculate a "returns" in the tripSummary function. sort of a "if the last position is further than X km from the colony, mark as "Not", and if its within that X km buffer mark as "Yes". Bu obviously that adds quite a bit of complication!

**Comment by MartinBeal** [comment]
That could work, but we would need to introduce a new argument "InnerBuff" to tripSummary too so people can set this as they need. Which would be redundant for users of tripSplit.

**Comment by MartinBeal** [comment]
Which I now realize is the reason for the title of this issue haha

**Comment by steffenoppel** [comment]
I am not sure why we should re-write one function just so that people could skip another preparatory function, which would require the unnecessary copy-pasting of half of one function (tripSplit) into another one (tripSummary)? Is there a problem of running already prepared data through the tripSplit function? If the data are already clean, then nothing should change, so why is this a problem?
